schemas/IDLs/org.gel.models.report.avro/5.0.0-SNAPSHOT/ReportVersionControl.avdl

```diff
@@ -0,0 +1,14 @@
+@namespace("org.gel.models.report.avro")
+/**
+This protocol defines the Participants and Pedigree
+*/
+protocol VersionControl {
+
+    record ReportVersionControl {
+
+        /** This is the version for the entire set of data models as referred to the Git release tag
+        */
+        string gitVersionControl = "4.2.0";
```

Comment:
should this be `string gitVersionControl = "5.0.0";` ?

Reply:
agree